src/vaadin-grid.html

```diff
@@ -427,6 +427,7 @@
         super.attributeChangedCallback(name, oldValue, newValue);
         if (name === 'dir') {
           this.__isRTL = newValue === 'rtl';
+          this._updateScrollerMeasurements && this._updateScrollerMeasurements();
```

Comment:
```suggestion
          this._updateScrollerMeasurements();
```

Not sure why we need this?